index.html

```diff
@@ -4,10 +4,45 @@
         <meta charset="utf-8">
         <title>Drayton Village Hall</title>
         <meta name="author" content="Shamitha Sudheer and Sharda Mahendra">
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" crossorigin="anonymous">
+        <link href="css/styles.css" rel="stylesheet">
     </head>
     <body>
-        <h1>Drayton village Hall</h1>
-        <img src="./Drayton_Hall_image.jpg" alt="Drayton village Hall">
+        <nav class="navbar navbar-expand-md navbar-dark bg-dark fixed-top">
+            <a class="navbar-brand" href="#">Drayton Village Hall</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
+              <span class="navbar-toggler-icon"></span>
+            </button>
+      
+            <div class="collapse navbar-collapse" id="navbarsExampleDefault">
+              <ul class="navbar-nav mr-auto">
+                <li class="nav-item active">
+                  <a class="nav-link" href="#">Home <span class="sr-only">(current)</span></a>
+                </li>
+                <li class="nav-item">
```

Comment:
we can add Book your event as well

Reply:
Agreed! We will check this in and make changes later